gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```

Comment:
Can you delete this file.

Reply:
You will need this file if you're planing to implement any CI/CD otherwise I can delete.